src/pymor/algorithms/rand_la.py

```diff
@@ -0,0 +1,350 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright 2013-2021 pyMOR developers and contributors. All rights reserved.
```

Comment:
You can drop the year range. We've dropped them everywhere recently.

Reply:
okay i will drop it, thanks.